docs/api/spaces-management/copy_saved_objects.asciidoc

```diff
@@ -55,13 +55,19 @@ You can request to overwrite any objects that already exist in the target space
   (Optional, boolean) Creates new copies of saved objects, regenerates each object ID, and resets the origin. When used, potential conflict
   errors are avoided. The default value is `true`.
 +
-NOTE: This cannot be used with the `overwrite` option.
+NOTE: This option cannot be used with the `overwrite` and `compatibilityMode` options.
 
 `overwrite`::
   (Optional, boolean) When set to `true`, all conflicts are automatically overridden. When a saved object with a matching `type` and `id`
   exists in the target space, that version is replaced with the version from the source space. The default value is `false`.
 +
-NOTE: This cannot be used with the `createNewCopies` option.
+NOTE: This option cannot be used with the `createNewCopies` option.
+
+`compatibilityMode`::
```

Comment:
That's a good question, and I'm not quite sure. I don't think there is any good reason why `import` and `copy` APIs accept these option differently.

The `import` API uses query string parameters because the body is the raw file content, so it's understandable why it uses both body and query string parameters. As for the `copy` API, having `spaces` and `objects` parameters in the body makes sense since they might be quite large potentially, but I'd rather use query string for the rest of the parameters to be consistent with the `import` API. I guess, we put everything in the body just to make it simpler for us :man_shrugging: